sdk/core/azure-core-http-netty/CHANGELOG.md

```diff
@@ -3,8 +3,12 @@
 ## 1.12.0-beta.1 (Unreleased)
 
 ### Features Added
-
+  
 ### Breaking Changes
+- The `NettyAsyncHttpClientProvider.createInstance()` now returns shared instance of `HttpClient`. Azure libraries that 
+  don't configure custom `HttpClientOptions` or `HttpClient` instance are now using single shared instance as well. 
+  A new configuration environment variable `AZURE_DISABLE_HTTP_CLIENT_SHARING` has been added for backwards 
```

Comment:
Doesn't it default to true if unset? So the normal configuration is to use previous behavior, and the only way to get new behavior is to set it to `false`?

Reply:
Yes. Was thinking of not breaking existing customers' behavior. So setting it to 'true' will make no change to the customer who running the code without break.

Reply:
Agreed we should default to true to maintain behavior. I think the doc here is confusing as it says to do the thing that happens automatically. I would state that setting to false enables new behavior, and leaving it unset or setting it to true maintains existing behavior.

Reply:
updated